assets/src/edit-story/output/page.js

```diff
@@ -44,6 +50,13 @@ function OutputPage({ page, autoAdvance, defaultPageDuration }) {
     ? `el-${longestMediaElement?.id}-media`
     : `${defaultPageDuration}s`;
 
+  const hasPageAttachment = page.pageAttachment?.url?.length > 0;
+
+  // Check for invalid elements, @todo this should come from the pre-publish checklist in the future.
+  const validElements = regularElements.filter(
+    (element) => hasPageAttachment && !isElementBelowLimit(element)
+  );
```

Comment:
Hmm, when trying this out in practice, I'm thinking that maybe we should just remove the link from the element (but leave it in place) rather than completely remove the element? Or is it better to remove it to make it clear to editors, that something is wrong?

Reply:
Yes, makes sense, let's just remove the link, that's what the warning message in the editor also indicates.

Changed.